app/services/synchronizer/marc_synchronizer.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+class Synchronizer
+  # Synchronizes an AbID to Alma.
+  # Finds an AbID's associated holding record and updates its 852h (call number)
+  # with the AbID.
+  class MarcSynchronizer
+    attr_reader :absolute_identifier
+    delegate :alma_item, to: :absolute_identifier
+    def initialize(absolute_identifier:)
+      @absolute_identifier = absolute_identifier
+    end
+
+    def sync!
+      update_holding_marc
+      url = "#{bibs_base_path}/#{mms_id}/holdings/#{holding_id}"
+      data = "<holding><record>#{holding_marc.to_xml}</record></holding>"
+      HTTParty.put(url, headers: headers, body: data)
+      absolute_identifier.sync_status = "synchronized"
+      absolute_identifier.save
+    end
+
+    private
+
+    def holding
+      @holding ||= Alma::BibHolding.find(mms_id: mms_id, holding_id: holding_id)
+    end
+
+    def holding_marc
+      @holding_marc ||= MARC::XMLReader.new(StringIO.new(holding["anies"][0])).to_a.first
+    end
+
+    def update_holding_marc
+      @modified_holding_marc ||=
+        begin
+          # There will always be an 852.
+          holding_marc.each_by_tag("852") do |field|
+            # Means "local"
+            field.indicator1 = "8"
```

Comment:
what does Means "local" mean?

Reply:
I guess it technically means "other scheme"? This is just what Don had said.